app/kumactl/pkg/install/k8s/sorter.go

```diff
@@ -1,21 +1,32 @@
 package k8s
 
 import (
-	helm_manifest "k8s.io/helm/pkg/manifest"
-	helm_releaseutil "k8s.io/helm/pkg/releaseutil"
+	"log"
+
+	"helm.sh/helm/v3/pkg/chartutil"
+	"helm.sh/helm/v3/pkg/releaseutil"
 
 	"github.com/kumahq/kuma/app/kumactl/pkg/install/data"
 )
 
 func SortResourcesByKind(files []data.File) []data.File {
 	singleFile := data.JoinYAML(files)
-	resources := helm_releaseutil.SplitManifests(string(singleFile.Data))
-	manifests := helm_manifest.SplitManifests(resources)
-	SortByKind(manifests)
+	resources := releaseutil.SplitManifests(string(singleFile.Data))
+
+	hooks, manifests, err := releaseutil.SortManifests(resources, chartutil.VersionSet{"v1", "v1beta1", "v1alpha1"}, releaseutil.InstallOrder)
+	if err != nil {
+		log.Fatal(err)
```

Comment:
this is the equivalent to `panic(err)`. Is there a reason you are not returning an error from `SortResourcesByKind `?

Reply:
Well, not really any reason. I just kept the interface the same and changed the implementation to rely on an existing Helm code. We can return errors of course. Let me try and see where this goes.